editor/assets/chunks/common/texture/texture-lod.chunk

```diff
@@ -46,6 +54,10 @@ vec4 fragTextureGrad (samplerCube tex, vec3 P, vec3 dPdx, vec3 dPdy) {
       return textureCube(tex, P);
     #endif
   #else
-    return textureGrad(tex, P, dPdx, dPdy);
+    #if defined(CC_USE_WGPU)
+      return textureLod(tex, P, 0.0);
```

Comment:
wgsl does not have `textureGrad`?

Reply:
> wgsl does not have `textureGrad`?
Yes, the problem occurred during the compilation with glslang.js.

Reply:
![image](https://github.com/cocos/cocos-engine/assets/13654656/e79add38-64a0-4178-89ba-96b7b5b70b4e)
wgsl has `textureSampleGrad`. It seems strange.